lib/impl/AzurePresignTvm.js

```diff
@@ -29,7 +29,10 @@ class AzurePresignTvm extends Tvm {
     this._addToValidationSchema('azureStorageAccount')
     this._addToValidationSchema('azureStorageAccessKey')
     this._addToValidationSchema('blobName')
-    this._addToValidationSchema('expiryInSeconds', joi.number().integer().min(30).max(18000).required())
+    this._addToValidationSchema('expiryInSeconds', joi.number().integer().min(2).max(86400).required())
```

Comment:
(24 * 60 * 60) is way clearer, also/or a comment saying `// limit expiry between 2 seconds and 24 hours`